src/reducers/entities/teams.js

```diff
@@ -64,7 +64,28 @@ function myMembers(state = {}, action) {
         }
         return nextState;
     }
+    case TeamTypes.RECEIVED_TEAMS_LIST: {
+        const nextState = {...state};
+        const teamsList = teamListToMap(action.data);
+        Object.keys(nextState).forEach((myMemberTeamId) => {
+            if (teamsList[myMemberTeamId] && teamsList[myMemberTeamId].delete_at) {
+                Reflect.deleteProperty(nextState, myMemberTeamId);
+            }
+        });
 
+        return nextState;
+    }
+    case TeamTypes.RECEIVED_TEAMS: {
+        const nextState = {...state};
+        const team = action.data;
+        Object.keys(team).forEach((teamId) => {
```

Comment:
Here you're looping through the teams you've received but in the above case you're looping through all of state. Is there a reason for doing it differently? I'd prefer to be consistent, leaning towards looping through the incoming data rather than the state as the incoming data is likely to be smaller

Reply:
For the RECEIVED_TEAMS_LIST, I just thought the incoming data would be bigger than the state that's why I did looping through state. In addition to your comment, the difference between the two might not really be significant, so consistency is the real deal.